gcp_variant_transforms/testing/integration/large_tests/platinum_with_merge.json

```diff
@@ -0,0 +1,23 @@
+{
+  "test_name": "platinum-with-merge",
+  "table_name": "platinum_with_merge",
+  "input_pattern": "gs://genomics-public-data/platinum-genomes/vcf/*.vcf",
+  "runner": "DataflowRunner",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "worker_machine_type": "n1-standard-16",
+  "max_num_workers": "10",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY"],
+      "expected_result": {"num_rows": 309551691}
```

Comment:
Actually, just noticed that this should be updated as it's from no-merge option.

Reply:
Right, forgot to update it and for this case it never reached table validation before. Thanks for pointing it out.